recipes/vcftools/0.1.11/build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
-
+set -euo pipefail
+sed -i.bak -e 's/DIRS = cpp perl/DIRS = cpp/' ./Makefile
+sed -i.bak -e "s/CFLAGS = /CFLAGS = -I$PREFIX\/include /" ./cpp/Makefile
```

Comment:
You need to use a different delimiter than `/` because the `$PREFIX` contains slashes as well. `'s|CFLAGS = |CFLAGS = ...|`, for example.